### DISCUSSION TITLE
Too many active WebGL contexts. Oldest context will be lost.

### DISCUSSION THREAD

**Discussion by Ysrbolles** [uber/react-map-gl#2187] [Q&A] [answered]
Hello Guys,

I'm Creating an app using `Mapbox` and I got this error on the map, after mounting the page the map disappears, can anyone tell me what's the problem?

https://github.com/visgl/react-map-gl/assets/37486496/298d49e7-3dfe-47ba-bb4d-08babd2d5fec

## console warnings:

<img width="630" alt="Screenshot 2023-05-31 at 11 13 44" src="https://github.com/visgl/react-map-gl/assets/37486496/ae14b492-b62f-4c6b-91b3-90b3bbe10fa2">

**Comment by Pessimistress** [comment]
Your context lost errors are from Three.js, not mapbox-gl.

**Reply by Ysrbolles** [reply]
@Pessimistress but why did the Map disappear?

**Reply by Pessimistress** [reply]
If other part of your application crashes WebGL, then Mapbox cannot function.

**Reply by Ysrbolles** [reply]
I was using  `<model-viewer>` at first, but I deleted everything related to it, and still the same problem, it's coming from `vendors-node_modules_mapbox-gl_dist_mapbox-gl_js.chunk.js`

@Pessimistress any idea Please 🙏🏻

<img width="910" alt="Screenshot 2023-06-01 at 18 13 50" src="https://github.com/visgl/react-map-gl/assets/37486496/d5ed22db-0cde-4ebe-b846-7fa61bf71c5c">

**Comment by Ysrbolles** [comment, marked as answer]
Solved! I noticed that I was using the "fingerprintjs" library, which uses WebGL to generate a unique fingerprint.
